src/SwedbankPay.Sdk/PaymentOrders/PaymentOrderResponseContainer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+using SwedbankPay.Sdk.Exceptions;
+
+namespace SwedbankPay.Sdk.PaymentOrders
+{
+    public class PaymentOrderResponseContainer
```

Comment:
What is the purpose of this class? Wouldn't these kind of methods belong on the response object itself?

Reply:
To make the serialization match the json request.